### PR TITLE
feat(workflow): Add alert + rule to `load-mocks`

### PR DESCRIPTION
**Pull request by billyvg**
This adds an alert and alert rule to `load-mocks` for local testing

### CODE REVIEW

File: bin/load-mocks [hunk 1]
```diff
@@ -662,7 +689,7 @@ def main(num_events=1, extra_events=False):
 
             if hasattr(buffer, "process_pending"):
                 print ("    > Processing pending buffers")  # NOQA
-                buffer.process_pending()
+                print ("    > Processing pending buffers")  # NOQA
```
Comment (wedamija) [inline bin/load-mocks]: What's this change?

Comment (billyvg) [reply bin/load-mocks]: hmmr term may have been bugging out, will fix

File: bin/load-mocks [hunk 2]
```diff
@@ -21,6 +21,15 @@ from pytz import utc
 
 from sentry import buffer, roles, tsdb
 from sentry.event_manager import HashDiscarded
+from sentry.incidents.logic import (
+    AlertRuleNameAlreadyUsedError,
+    create_alert_rule,
+    create_alert_rule_trigger,
+    create_incident,
+    create_incident_activity,
+    create_initial_event_stats_snapshot,
```
Comment (wedamija) [inline bin/load-mocks]: These are both unused

**Review by wedamija — APPROVED** [review]

File: bin/load-mocks (hunk 2 above)
Comment (billyvg) [reply bin/load-mocks]: Dang, does the linter not lint this file?

Comment (billyvg) [reply bin/load-mocks]: I guess they are warnings

**Pull request merged**